www/caddy/+POST_INSTALL.post

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# Define the path to the +TARGETS file
+TARGETS_FILE="/usr/local/opnsense/service/templates/Pischem/Caddy/+TARGETS"
+
+# Trigger a reload of the Caddy template to ensure all necessary files are generated
+echo "reload configuration templates"
+configctl template reload Pischem/Caddy
+
+# Trigger a syslog-ng restart since caddy generated a configuration file for it
+echo "restart syslog-ng"
+configctl syslog restart
+
+# Define the path to the setup script
+echo "executing setup script"
+SETUP_SCRIPT="/usr/local/opnsense/scripts/Pischem/Caddy/setup.sh"
```

Comment:
unintended side effect. the first start will take care of it

Reply:
same here